### ISSUE TITLE
Incorrect Expected name of type as first argument

### ISSUE THREAD

**Issue by albireox** [microsoft/pyright#808]
**Describe the bug**

I get a `Expected name of type as first argument` while defining a class using `type` even if I'm passing a string as the first argument.

**To Reproduce**

This code produces the error, where `CatalogdbModel` is a peewee model

```python
        rel_model = database.models[fname]
        model_name = 'CatalogTo' + rel_model.__name__

        class Meta:
            table_name = rtname
            primary_key = False

        RelationalModel = type(model_name, (CatalogdbModel,), {'Meta': Meta})
```

**Expected behavior**

No syntax error.

**VS Code extension or command-line**

Using pylance 2020.6.1.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.52, which I just published. It will be included in the next published version of Pylance.
